blue_green_deploy.go

```diff
@@ -48,8 +55,39 @@ func (p *BlueGreenDeploy) DeleteAllAppsExceptLiveApp(appName string) {
 
 }
 
-func (p *BlueGreenDeploy) PushNewApp(appName string, route plugin_models.GetApp_RouteSummary, manifestPath string) {
+func (p *BlueGreenDeploy) GetScaleParameters(appName string) AppScaleParameters {
+	appModel, err := p.Connection.GetApp(appName)
+	if err != nil {
+		return AppScaleParameters{}
+	}
+	appScaleParameters := AppScaleParameters{
+		InstanceCount: appModel.InstanceCount,
+		Memory:        appModel.Memory,
+		DiskQuota:     appModel.DiskQuota,
+	}
+	return appScaleParameters
+}
+
+func appendScaleArguments(args []string, scaleParameters AppScaleParameters) []string {
```

Comment:
Will use scaleParams instead for consistency since it is shorter (also AppScaleParameters -> ScaleParams).